examples/qml_features/cpp/main.cpp

```diff
@@ -9,12 +9,15 @@
 #include <QtQml/QQmlApplicationEngine>
 
 #include "custom_object.h"
+#include "cxx-qt-gen/rust_invokables.cxxqt.h"
 
 int
 main(int argc, char* argv[])
 {
   QGuiApplication app(argc, argv);
 
+  RustInvokables invokables(0.f, 0.f, 1.f, nullptr);
```

Comment:
Would be better to create a separate constructor.rs example in qml_features, then keep RustInvokables as it was.

Otherwise people searching through the qml_features examples won't be able to find custom constructors easily.

Reply:
I've migrated to implementing this on the two types that currently need a `Component.onComplete` callback, so that it's actually useful.
Hopefully that'll be easy enough to find.
We can always add a separate `Constructor` example later.